bundles/AdminBundle/Resources/public/js/pimcore/object/classes/class.js

```diff
@@ -1083,22 +1083,30 @@ pimcore.object.classes.klass = Class.create({
             value: data.index_key
         };
 
+        //fixes data to match store model
+        var indexesArray = [];
```

Comment:
```suggestion
        let indexesArray = [];
```

Reply:
Why not `const`?
```suggestion
        const indexesArray = [];
```

Reply:
@jdreesen We can also use `const`

Reply:
@torqdev Could you please take a look at this? Thanks

Reply:
@torqdev you know that you can just accept the suggestion, right? There's no need to manually create a commit ;)